index.js

```diff
@@ -161,7 +161,18 @@ precinct.paperwork = (filename, options = {}) => {
   const deps = precinct(content, options);
 
   if (!options.includeCore) {
-    return deps.filter((dep) => dep.startsWith('node:') ? false : !natives[dep]);
+    return deps.filter((dep) => {
+      if (dep.startsWith('node:')) {
+        return false
+      }
+
+      if (["test"].includes(dep)) { // some builtins can only be access via node: prefix
```

Comment:
Do we need this condition? It seems like we want require("node:test") to be excluded but require("test") to be included, which would already happen if you remove this condition.

I might not be following the nuance of the test builtin.

Reply:
If you includeCore = true, then we wouldn't run this filtering, which means require("node:test") would be returned as a dependency no?.

Reply:
It would be excluded, because it's part of the `natives` object. The "weird" thing about the new `test` builtin is that, although it's included in `process.binding("natives")` as `"test"`, it can only be required as `"node:test"`.

So this condition is necessary, yes.